src/serac/physics/utilities/variational_form/CMakeLists.txt

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) 2019-2020, Lawrence Livermore National Security, LLC and
+# other Serac Project Developers. See the top-level LICENSE file for
+# details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+
+set(weakform_depends serac_physics mfem mpi)
+
+#if(ENABLE_CUDA)
+#    list(APPEND weakform_depends cuda)
+#endif()
+
+# Add the library first
+set(weakform_headers
+    weak_form.hpp
+    tensor.hpp
+    dual.hpp
+    tuple_arithmetic.hpp
+    polynomials.hpp
+    quadrature.hpp
+    finite_element.hpp
+    detail/hexahedron_h1.inl
+    detail/hexahedron_hcurl.inl
+    detail/quadrilateral_h1.inl
+    detail/quadrilateral_hcurl.inl
+    detail/quadrilateral_L2.inl
+    detail/segment_h1.inl
+    detail/metaprogramming.hpp
+    )
+
+blt_add_library(
+      NAME        serac_weakform
+      HEADERS     ${weakform_headers}
+      DEPENDS_ON  ${weakform_depends}
+      )
+
+
+install(FILES ${weakform_headers} DESTINATION include/serac/physics/utilities/weak_form )
```

Comment:
We should be consistent one way or the other or immediately do the rename PR to functional.  

`serac/physics/utilities/variational_form` is the repo dir then you are installing into `include/serac/physics/utilities/weak_form`

Reply:
I'll make everything use `weak_form` for consistency now, and we'll rename to `functional` in the future. Does that work?